kubetest/gke.go

```diff
@@ -320,6 +323,15 @@ func (g *gkeDeployer) Up() error {
 		}
 
 	}
+	if g.nodeLocations != "" {
```

Comment:
Can't you just replace additionalZones with nodeLocations (since the former seems to be deprecated for the latter)? Or you plan to do that as next stage?

Reply:
Done.